go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/jaypipes/ghw v0.8.1-0.20210827132705-c7224150a17e
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.12
 	github.com/klauspost/cpuid/v2 v2.0.11
-	github.com/onsi/ginkgo v1.14.0
+	github.com/onsi/ginkgo v1.16.5
```

Comment:
Updating the dependencies would be better to do in a separate PR. Also, let's keep these in sync with k/k

Reply:
done